ReferenceError: artifacts is not defined

I've just cloned the repository, installed the dependencies and I'm getting this error when I run `npm test`:

```
[lodestar_chain](master)$ npm test

> lodestar-chain@0.1.0 test /home/hugo/workspace/Sharding/lodestar_chain
> mocha --recursive

/home/hugo/workspace/Sharding/lodestar_chain/test/test_validator_registration.js:1
(function (exports, require, module, __filename, __dirname) { var ValidatorRegistration = artifacts.require('ValidatorRegistration');
                                                                                                    ^

ReferenceError: artifacts is not defined
    at Object.<anonymous> (/home/hugo/workspace/Sharding/lodestar_chain/test/test_validator_registration.js:1:101)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at /home/hugo/workspace/Sharding/lodestar_chain/node_modules/mocha/lib/mocha.js:250:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/home/hugo/workspace/Sharding/lodestar_chain/node_modules/mocha/lib/mocha.js:247:14)
    at Mocha.run (/home/hugo/workspace/Sharding/lodestar_chain/node_modules/mocha/lib/mocha.js:576:10)
    at Object.<anonymous> (/home/hugo/workspace/Sharding/lodestar_chain/node_modules/mocha/bin/_mocha:637:18)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:607:3
npm ERR! Test failed.  See above for more details. 
```

I couldn't find information about this error which is related to mocha, all the information I found is related to truffle.

--- a/README.md
+++ b/README.md
@@ -10,5 +10,10 @@ You will need to go over the [specification](https://notes.ethereum.org/SCIg8AH5
 
 In order to run the code in this repository, you will first need to run `npm install` to install all dependencies. Then, to run the tests, you will need to run `npm test`
 
+## Note about tests
+`npm test` runs `truffle test` please ensure that there is a terminal window with ganache-cli running
+
+Note: There is a WIP that will auto boot a ganach terminal wehn you run `npm test` :)
+
 ## Contributors
 If you would like to contribute, please submit an issue or talk to us on our [gitter](https://gitter.im/chainsafe/lodestar-chain).

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
   "homepage": "https://github.com/ChainSafe/lodestar_chain#readme",
   "main": "index.js",
   "directories": {
-    "test": "test"
+    "test": "truffle test"
   },
   "scripts": {
     "test": "mocha --recursive"

